java/client/src/org/openqa/selenium/remote/Command.java

```diff
@@ -45,7 +45,7 @@ public String getName() {
   }
 
   public Map<String, ?> getParameters() {
-    return parameters == null ? new HashMap<String, Object>() : parameters;
+    return parameters == null ? new HashMap<>() : parameters;
```

Comment:
From travis-ci:

1. ERROR in /home/travis/build/SeleniumHQ/selenium/java/client/src/org/openqa/selenium/remote/Command.java (at line 48)
	return parameters == null ? new HashMap<>() : parameters;
	       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Type mismatch: cannot convert from Map<capture#5-of ? extends Object,capture#6-of ? extends Object> to Map<String,?>